workspaces/pwa.md

```diff
@@ -21,36 +19,3 @@ To use Coder as a PWA, you must be using Google Chrome or Microsoft Edge.
 1. Follow your browser's instructions for installing the application as a PWA.
 
 Please note that applications are currently installed on a per-workspace basis.
-
-## Coder PWA for v. 1.14.x or Earlier
-
-> The following instructions are applicable only if you're using Coder v 1.14.x
-> or earlier.
-
-## Chrome installation
-
-You can install Coder as a PWA when using Chrome in one of three ways:
-
-1. Click **Install App**, which is located at the top of the Coder Dashboard.
-1. Go to your **Settings** menu, and select **Install Coder**.
-1. Click the **plus** icon located in the browser's URL bar.
-
-### Uninstalling the Chrome progressive web app
-
-To remove the Coder progressive web app at any time, go to **Settings** >
-**Uninstall Coder**. You'll be prompted to remove Coder; confirm by clicking
-**Remove**. You can also check the box to clear Coder data if desired.
-
-## Edge installation
-
-You can install Coder as a PWA when using Edge in one of three ways:
-
-1. Click **Install App**, which is located at the top of the Coder Dashboard.
-1. Click the **plus** icon located in the browser's URL bar.
-1. Go to Settings > Apps and select **Install Coder**.
-
-### Uninstalling the Edge progressive web app
-
-To remove the Coder progressive web app, go to **Settings** > **Uninstall
-Coder**. You'll be prompted to remove Coder; confirm by clicking **Remove**. You
-can also check the box to clear Coder data (if desired).
```

Comment:
any reason why we're removing this documentation?

Reply:
@ericpaulsen Yes, it's out of date, but with versioning, you can still access it (though no reason for recent versions to see)